gobblin-yarn/src/main/java/org/apache/gobblin/yarn/YarnService.java

```diff
@@ -453,7 +453,7 @@ private EventSubmitter buildEventSubmitter() {
   /**
    * Request an allocation of containers. If numTargetContainers is larger than the max of current and expected number
    * of containers then additional containers are requested.
-   *
+   * <p>
```

Comment:
Dumb question, but does <p> need a closing tag? The placement of these tags make it seem like you meant to do <br>

Reply:
As per https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html , it does not need a closing tag. It's more like a paragraph separator than a paragraph limits definer. I just followed what IntelliJ suggested.